client/src/pages/Profil/Profil.jsx

```diff
@@ -3,7 +3,7 @@ import * as Yup from "yup";
 import { useOutletContext, Link } from "react-router-dom";
 
 //RTK Query
```

Comment:
Bro, please use good spelling in the code. It is important for later. This is just one of many. You wrote Profil instead of Profile.

Reply:
I will fix that in the next pull request